news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.7.6"
+subtitle: "Maintenance version of the 1.7.7.x branch"
+date:   2021-08-06 12:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2021/08/build_1.7.7.6.png
+icon: icon-leaf
+tags:
+ - version
+ - maintenance
+ - patch
+ - releases
+ - 1.7.7.x
+ - 1.7
+---
+
+PrestaShop 1.7.7.6 is now available. This maintenance release fixes 27 regressions reported since the 1.7.7.0, including 2 critical regressions.
+
+![1.7.7.6 is available!](/assets/images/2021/08/build_1.7.7.6.png)
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don't forget to backup before.
+
+## Critical fixes
+
+The bug affects critical functionality or critical data and there is no workaround to avoid it. A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Two critical bugs have been fixed in 1.7.7.6:
+
+- [Orders page - an exception is displayed when deleting a customer or a guest](https://github.com/PrestaShop/PrestaShop/issues/25319)
+- [Smarty template isCached function bug](https://github.com/PrestaShop/PrestaShop/issues/24235)
+
+## Main fixes
```

Comment:
"Main fixes" seems to indicate there are "main" fixes and "other" fixes. But there is no other fixes, if I am not wrong ?

Then maybe we can simply write
```suggestion
## Bug fixes list
```

Reply:
The others are the critical ones

Reply:
```suggestion
## Other bug fixes
```